src/main/java/seedu/address/ui/ContactsClearConfirmWindow.java

```diff
@@ -14,12 +14,13 @@
 /**
  * Controller for a clear confirmation
  */
-public class ConfirmWindow extends UiPart<Stage> {
+public class ContactsClearConfirmWindow extends UiPart<Stage> {
```

Comment:
Since the `ContactsClearConfirmWindow` and `MeetingsClearConfirmWIndow` have exactly the same UI, why don't we combine them into one? Is it possible to pass in a `boolean` variable indicating whether the contacts should be cleared or not to a method to show the `ConfirmWindow`, so that the `ConfirmWindow` can show itself instead of the `MainWindow` calling `show()` on it?

Just a suggestion since I made the `ConfirmWindow` with reusability in mind, but if it's too much work then you can ignore it.

Reply:
Yup, I have made the requested changes. Hope you take a look at it

Reply:
A little problem is that I made a global boolean variable just `private` in order to set the bool variable passed in as the variable so the immutability of the system diminishes a bit.